File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Firefox()

wait = WebDriverWait(driver, 10)

driver.get("https://training-support.net/webelements/alerts")

print("Page title:", driver.title)

driver.find_element(By.ID, "prompt").click()

alert = wait.until(EC.alert_is_present())

print("Alert text:", alert.text)

alert.send_keys("Awesome!")

alert.accept()

print("Result:", driver.find_element(By.ID, "result").text)

driver.quit()

File: Selenium/Activities/Activity20.java
package examples;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity20 {

    public static void main(String[] args) {

        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/alerts");

        System.out.println("Page title: " + driver.getTitle());

        driver.findElement(By.id("prompt")).click();

        Alert alert = driver.switchTo().alert();

        System.out.println("Alert text: " + alert.getText());

        alert.sendKeys("Awesome!");

        alert.accept();

        System.out.println("Result: " + driver.findElement(By.id("result")).getText());

        driver.quit();
    }
}
